doc/recipes/flavor_registry/controls.md

```diff
@@ -0,0 +1,91 @@
+# Touch Controls
+
+## Joystick
+
+Please reference the [Joystick documents](../../flame/inputs/other-inputs.md) for a complete example.
+
+## Touch Controls within the HUD
+
+```dart
```

Comment:
Is this to be embedded as a running example too?
Could we reference the dart file code for that example instead of having it in the markdown then maybe? Since if anything changes in Flame these snippets easily become outdated.

Reply:
Yes.  I was going to get around to doing this as an embed as well.  Just wanted to see your thoughts on what I had so far and direction.

Reply:
I just need to add this embed.  I will get to this tonight.

Reply:
Do let me know if you think we can augment the `flutter_app` extension in any way, to accommodate the recipes: adding a new mode, or changing the style, etc.

Reply:
So far, everything just works really well. I haven't found anything that I wasn't able to do. I will take a closer look at the parent and see if there is anything I think could be added. I will say it's been pretty cool because it just works!